### PR TITLE
Change import users from csv to generate username if it's missing

### PR DESCRIPTION
**Pull request by karenc**
This is for importing users from legacy osc site.

### CODE REVIEW

File: lib/import_users.rb [hunk 1]
```diff
@@ -56,7 +58,13 @@ def read
   end
 
   def create_user(username, password_digest, title, first_name, last_name, full_name, email_address)
-    person = Person.create!
+    # Check whether the user is already in the database
+    ea = EmailAddress.find_by_value(email_address)
```
Comment (jpslav) [inline lib/import_users.rb]: should we limit to just verified email addresses?  `EmailAddress.verified.where{value: email_address}`

File: lib/import_users.rb [hunk 2]
```diff
@@ -86,4 +94,24 @@ def create_user(username, password_digest, title, first_name, last_name, full_na
     ea.verified = true
     ea.save!
   end
+
+  def generate_username(first_name, last_name, email)
+    # Use first name and last name as the basis for generating username
+    base = "#{first_name} #{last_name}".strip
```
Comment (jpslav) [inline lib/import_users.rb]: is there a reason for the space between first and last name?  gets `gsub`bed out later.

File: lib/import_users.rb (hunk 1 above)
Comment (jpslav) [inline lib/import_users.rb]: if we don't use verified emails, there's an outside chance that we'd link different people together.

**Comment by jpslav** [comment]
Review complete

**Comment by karenc** [comment]
Updated.

**Pull request merged**